### ISSUE TITLE
Missing dot on float formatting 

### ISSUE THREAD

**Issue by venth** [DataDog/java-dogstatsd-client#24]
Currently float formatting is based on user locale and sometimes send comma instead of dot. This issue is already fixed in the original project.

Please merge pull request (issue) https://github.com/indeedeng/java-dogstatsd-client/issues/3

**Comment by masci** [comment]
Hi @venth ,
the commit is already here: https://github.com/DataDog/java-dogstatsd-client/commit/d445d745632fb6f96df366c4cef4b70477a173ba#diff-f4b9746cffe53d8778d5b04a486a2b2d you can check out the diff.
Maybe it's a regression, are you experiencing the very same issue as the original one?

**Comment by venth** [comment]
Yes. I've noted that server uses polish locale, which causes error during counter increment send to statsd. Formatting need to work regardless user locale. I browsed through issues of new and old project and found fitting one.
